linux/step05_centos7_nginx.sh

```diff
@@ -1,15 +1,19 @@
 #!/bin/bash
 
+#start-copy
+cp setup_omero_nginx.sh ~omero
+#end-copy
+
 #start-install
 yum -y --enablerepo=cr install nginx
```

Comment:
why do we need cr? EPEL is installed in https://github.com/ome/omero-install/pull/94/files#diff-2722b62c943402fa409b2ef40375cf5dR3

from EPEL you also install nginx/1.6.3